sdk/servicebus/test-resources.json

```diff
@@ -227,10 +197,6 @@
       "type": "string",
       "value": "[parameters('testApplicationId')]"
     },
-    "AZURE_CLIENT_SECRET": {
```

Comment:
Do we need to set AZURE_TENANT_ID and such if the environment already specifies `AZURESUBSCRIPTION_CLIENT_ID`, `AZURESUBSCRIPTION_TENANT_ID`

Reply:
Per grove migration guideline - only *_CLIENT_SECRET needs  to be removed. Based on this conversation, I tried running CI test after removing client_id and tenant_id from test-resources.json, and the tests passed. 

But I'm not sure if they are needed for other use cases like `-userAuth`. I'll follow up with grove on client_id|tenant_id, and if needed update the migration guideline / will make a follow up pr  accordingly.